examples/jsm/controls/OrbitControls.js

```diff
@@ -224,8 +224,28 @@ class OrbitControls extends EventDispatcher {
 
 				spherical.makeSafe();
 
+				if ( scope.enableDamping === true ) {
+
+					if ( scale !== 1 ) {
+
+						zoomFrom = spherical.radius;
+						zoomTo = spherical.radius * scale;
+						zoomDelta = ( zoomTo - zoomFrom ) * 3; // x3 because slow
```

Comment:
Is this magic number hardware-specific?

Reply:
Tested on windows, "x3" is chosen to make the dollying more comfortable ~

Reply:
I'm not sure about the mechanism of macOS's built-in damping, does it means that 'handleMouseMoveDolly' is called multiple times during a single mouse wheel movement?

Reply:
No. In MacOS, scrolling inertia is built into the OS. This PR is messing that up.